jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/configuration/CreateJvmRunSteps.java

```diff
@@ -99,4 +106,8 @@ public void validateJvm(final Map<String, String> jvmDetails) {
     public void waitForJvm(String name) {
         jwalaUi.waitUntilElementIsVisible(By.xpath("//button[text()='" + name + "']"));
     }
+
+    private String getPropertyValue(final String key) {
+        return paramProp.getProperty(key) == null ? key : paramProp.getProperty(key);
```

Comment:
actually no....imagine in case of sorting I have a hostname as aaaHost, in this case I would use this itself as the properhost name since there is no value present in the properties file